.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: true
 language: java
-install: gradle clean 
+install: ./gradlew clean
```

Comment:
this is causing build failures in travis

Reply:
My mistake, I didn't realize the .gitignore was excluding all jar files.

The reason of this change is that the current pipeline is wrong. The code specifies gradle version 2.3, but using _gradle_ command it uses whichever version is installed into the travis container (4.X). So that produces a mismatch, and therefore possible building issues.

I excluded the gradle-wrapper.jar from the ignored list and now everything runs properly.

This change is done to use only the version specified